extensions/azurePublishNew/src/components/provisioningWizards/CreateResourcesWizard.tsx

```diff
@@ -47,6 +49,7 @@ export const CreateResourcesWizard = React.memo((props: Props) => {
           <ResourceConfigurationStep onResourceConfigurationChange={setIsValidResourceConfiguration} />
         ),
         navigationState: { canGoNext: isValidResourceConfiguration },
+        onBack: () => persistResourceConfiguration(),
```

Comment:
It seems strange to be persisting resource configuration during onBack.  Is this how you keep state between the sub-wizards?  My assumption was that recoil would be holding the current state for all the steps while the wizard was running, the user could freely move forward and back, and when the wizard complete it would save the state for provisioning to pick up.

Reply:
We use recoil to keep state between wizard steps, local storage is used to stash the wizard state when exited from the extension and restore it, when the extension loads the next time.